XiEditor/CoreConnection.swift

```diff
@@ -39,24 +39,37 @@ struct FileWriter {
             return nil
         }
     }
-
```

Comment:
There's a bunch of whitespace changes in this file that should be cleaned up. Xcode likes to introduce them. 😕